docs/workflow/ci/triaging-failures.md

```diff
@@ -0,0 +1,202 @@
+For Docker environments, the Docker container mapping for coreclr runs is defined [here](../../../eng/pipelines/coreclr/templates/helix-queues-setup.yml)
+and for libraries runs is defined [here](../../../eng/pipelines/libraries/helix-queues-setup.yml).
+
+## Test configuration
+
+Many test runs use a non-default product configuration, to allow re-using existing test assets to stress various aspects of the system.
+Determine the precise test configuration under which the test has failed. This might be evident from the test job name. For example,
+`net7.0-windows-Release-x86-CoreCLR_checked-jitstress1-Windows.10.Amd64.Open` is a libraries test run on Windows with a Release x86 libraries
+build, Checked coreclr build, and setting the `COMPlus_JitStress=1` configuration setting, in the `Windows.10.Amd64.Open` Helix queue.
+
+You need to be careful when reproducing failures to set all the correct environment variables. In the above example, if you look at the
+test failure console log, you find:
+
+```
+C:\h\w\AE88094B\w\B1B409BF\e>set COMPlus 
+COMPlus_JitStress=1
+COMPlus_TieredCompilation=0
+```
+
+Thus, you can see that you also need to set `COMPlus_TieredCompilation=0` when attempting to reproduce the failure.
+
+On non-Windows platforms, you'll see a similar output for the test configuration. E.g.,
+
+```
++ printenv
++ grep COMPlus
+COMPlus_TieredCompilation=1
+COMPlus_JitStress=1
+COMPlus_DbgMiniDumpName=/home/helixbot/dotnetbuild/dumps/coredump.%d.dmp
+COMPlus_DbgEnableMiniDump=1
+```
+
+You might need to set variables in addition to the `COMPlus_*` (equivalently, `DOTNET_*`) variables. For example, you might see:
+```
+set RunCrossGen2=1 
+```
+which instructs the coreclr test wrapper script to do crossgen2 compilation of the test.
+
+Similarly,
+```
+set RunningIlasmRoundTrip=1
+```
+triggers an ildasm/ilasm round-trip test (that is, the test assembly is disassembled, then re-assembled, then run).
+
+And,
+```
+set DoLink=1
+```
+triggers ILLink testing.
+
+## Product and test assets
+
+To reproduce and/or debug a test failure, you'll need the product and test assets. You can either build these using the normal build processes,
+or you can download the ones used by the CI system. Building your own is often preferable so you can build a Debug flavor for better
+debugging fidelity: the CI typically runs with Checked and sometimes Release components.
+
+If downloading assets from the CI, it's often easiest to use the `runfo` tool:
+- [runfo Website](https://runfo.azurewebsites.net)
+- [runfo Documentation](https://github.com/jaredpar/runfo/tree/master/runfo#runfo)
+
+To install `runfo` as a .NET CLI global tool:
+
+```sh
+dotnet tool install --global runfo
+dotnet tool update --global runfo
+```
+
+Then use the `runfo get-helix-payload` command with the job name and workitem name. You can get these from the `Debug` tab of the
```

Comment:
worth mentioning that it's also in the top line of the console log? At least, the job ID and work item ID are. But apparently runfo requires the work item name, not ID? ID is a guid.